### PR TITLE
STCOM-442: Update Pane storybook examples

### PR DESCRIPTION
**Pull request by rasmuswoelk**
**Ticket:** https://issues.folio.org/browse/STCOM-442

In this PR I have made some changes for the`<Pane>` documentation and examples.

- Split Pane-readme up into more readme's. Now the underlying Pane "building blocks" have their own readme within their own component-folder
- Added examples for PaneBackLink and PaneCloseLink
- Added readme for PaneHeader with improved documentation for actionMenu etc.

Hopefully the updated documentation/examples will make it easier for 3rd party developers to utilize the `<Pane>` and related components to compose panesets etc.

image

### CODE REVIEW

**Comment by cherewaty** [comment]
Did you mean to delete `yarn.lock`?

File: lib/PaneBackLink/readme.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+Use with the `firstMenu` prop of a [`<Pane>`](/?selectedKind=Pane) or [`<PaneHeader>`](/?selectedKind=Pane&selectedStory=PaneHeader):
 
 ```
 <Pane firstMenu={<PaneBackLink to="/my-module-root" />}>
 ...
 </Pane>
 ```
 
-`<PaneBackLink>` passes along all received props to its child `<IconButton>`.
+`<PaneBackLink>` passes along all received props to its child [`<IconButton>`](/?selectedKind=IconButton).
```
Comment (cherewaty) [inline lib/PaneBackLink/readme.md]: These types of links work in the context of Storybook, but not when viewing the readme on GitHub.

Comment (rasmuswoelk) [reply lib/PaneBackLink/readme.md]: @cherewaty Hmm. Yeah that's a problem.

Either I remove the links or I can link to either the online version of the storybook (currently located at ux.folio.org/storybook) or to the component readme in the master branch.

What do you think?

Comment (cherewaty) [reply lib/PaneBackLink/readme.md]: Hmmm I think I would vote to make them absolute links to the component readme in the master branch. Not ideal, but I guess we have to pick one of the places as canonical 🤷‍♂️

**Comment by rasmuswoelk** [comment]
Decided to remove any links in readme's that were referencing storybook URL's.

Att. @cherewaty

**Pull request merged**